style/forms/ToggleButton.scss

```diff
@@ -0,0 +1,43 @@
+.ds-form-control-toggle {
+  position: relative;
+  align-items: center;
+  display: inline-flex;
+
+  input {
+    display: none;
+  }
+
+  &__slider {
+    position: relative;
+    overflow: hidden;
+    // float: left;
+    border-radius: 37px;
+    cursor: pointer;
+    width: 26.4px;
+    height: 16px;
+    background-color: #d7d7de;
+    transition: 0.4s;
+    margin-right: 4px;
+
+    input:checked + & {
+      background-color: $ds-brand-medium-color;
+    }
+
+    &:before {
+      border-radius: 37px;
```

Comment:
Because this element has same width and height, you can write `50%`

Reply:
done